### PR TITLE
Implement the 'actions' submodule

### LINKED ISSUE #85: Add support for GitHub Actions
The goal would be that bots written using gidgethub could driven by a GH Action by simply making the action trigger from a webhook event. That way a preexisting gidgethub bot run from a GH Action would be straightforward port. Hopefully this could eventually be as simple as having a template repo which contains everything needed to trigger the dispatch and users would just need a simple way to register their routes and edit the action YAML file to list the appropriate webhook triggers.

See https://github.com/actions/toolkit as necessary for how things work.

# [Logging commands](https://help.github.com/en/articles/development-tools-for-github-actions#logging-commands)
- [x] `set-env`
- [x] `set-output`
- [x] `add-path`
- [x] `debug`
- [x] `warning`
- [x] `error`
- [x] `add-mask`
- [x] `stop-commands`

# [Environment](https://help.github.com/en/articles/virtual-environments-for-github-actions)
- [x] Read webhook event file (`GITHUB_EVENT_PATH`)
- [x] `pathlib` object for workspace (`GITHUB_WORKSPACE`)

### PR DESCRIPTION
**Pull request by brettcannon**
Provide support for GitHub Actions as appropriate.

Closes #85

### CODE REVIEW

File: Third-Party Notice - GitHub Actions Toolkit.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright 2019 GitHub
```
Comment (webknjaz) [inline Third-Party Notice - GitHub Actions Toolkit.md]: Why is this needed?

Comment (brettcannon) [reply Third-Party Notice - GitHub Actions Toolkit.md]: Because I directly lifted implementation details from their JavaScript code to make sure I got the escaping correct as it isn't documented and it actually varies based on which part of the workflow command you execute. So, to make sure I cover my bases, I included that code's license as a 3rd-party notice.

Comment (webknjaz) [reply Third-Party Notice - GitHub Actions Toolkit.md]: Ah, I think I just read this
> permission notice shall be included in all copies or **substantial portions** of the Software

as a few mappings ported (not really copied) as non-substantial. But maybe you're right.

Either way, just by looking at the file name it's not very obvious what it relates to. I've seen some projects dedicating a separate folder (licensing?) for such things and mentioning the detail in README (maybe even README of that folder).

Comment (brettcannon) [reply Third-Party Notice - GitHub Actions Toolkit.md]: I've expanded in the file a bit to explain it better.
